cms_delete_content/controllers/main.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from openerp import http, _
+from openerp.http import request
+import json
+import werkzeug
+
+
+class DeleteMixin(object):
+
+    delete_confirm_template = 'cms_delete_content.delete_confirm'
```

Comment:
I use mixins exactly for this reason: it's kind of weird how inheritance works with controllers.